Sources/TextView/Representable.swift

```diff
@@ -30,7 +30,7 @@ extension TextView {
         }
 
         func updateUIView(_ view: UIKitTextView, context: Context) {
-            context.coordinator.update(representable: self)
+            context.coordinator.update(representable: self, sourceTextView: view)
```

Comment:
I wouldn't recommend this approach. I also wonder if actually a nice 'feature' here could be to add `selectedRange` as a property to the `TextView` to allow the `init` to accept this as well, this would allow pre-selection of text ranges as a part of the public API.

Thoughts @teffi ?